mudlet-mapper.xml

```diff
@@ -5815,6 +5515,229 @@ end</script>
                     <string>gmcp.Room.Info</string>
                 </eventHandlerList>
             </Script>
+            <Script isActive="yes" isFolder="no">
+                <name>Wings and Bixes</name>
+                <packageName></packageName>
+                <script>local exitsCreated = exitsCreated or {}
+
+local function tempSpecialExit(destination, command)
+  table.insert(exitsCreated, destination)
+  addSpecialExit(mmp.currentroom, destination, command)
+end
+
+function mmp.addWings()
+--Trimmed down version of the original function, meant to only be used for wings. (Where the command varies based on several MCONFIGS)
+  local function getcmd(word)
+    return
+      mmp.settings.removewings and
+      [[script:sendAll(&quot;wear wings&quot;, &quot;say *]] ..
+      (mmp.settings.winglanguage and mmp.settings.winglanguage or &quot;&quot;) ..
+      [[ ]] ..
+      word ..
+      [[&quot;, &quot;remove wings&quot;, false)]] or
+      [[script:send(&quot;say *]] ..
+      (mmp.settings.winglanguage and mmp.settings.winglanguage or &quot;&quot;) ..
+      [[ ]] ..
+      word ..
+      [[&quot;, false)]]
+  end
+
+  --creates a special exit from your current room to a destination, and adds it to a table so it can be easily cleared later.
```

Comment:
I wonder if it's not better to use one event handler per game instead of having one big for each.